### ISSUE TITLE
Computing hippocampal_volume / TIV

### ISSUE THREAD

**Issue by Metasoliton** [ANTsX/ANTs#526]
I have already performed hippocampal subfields segmentation via JLF, using the 5 CobraLab atlases.
Now I need to compute hippocampal volume by total intracranial volume, for each subject. I tried adding the subfields and thresholding but that did not produce great results across all subjects.

May I suggest adding a wiki-page on the simplest way to derive hippocampal volume and TIV, using ANTs?

**Comment by ntustison** [comment]
> Now I need to compute hippocampal volume by total intracranial volume, for each subject. I tried adding the subfields and thresholding but that did not produce great results across all subjects.

Can you elaborate as this description is confusing?  I don't understand how you're trying to "compute hippocampal volume by total intracranial volume"?

**Comment by Metasoliton** [comment]
Yes of course; I guess I should have explicitly stated "divided by". That is, one proposed measure of 'brain reserve' in the context of Alzheimer's disease is 'hippocampal volume divided by total intracranial volume' (lets call it HV / TICV). I was hoping I could use ANTs (instead of Freesurfer) to compute it.

**Comment by ntustison** [comment]
Brain volume:  [``antsBrainExtraction.sh``](https://github.com/ANTsX/ANTs/blob/master/Scripts/antsBrainExtraction.sh). 

If you used JLF to compute subfield labels, you should have the same labels in your target image that you do in your atlas images.  You should then be able to use ``ThresholdImage`` or ``ImageMath ... ReplaceVoxelValue`` to combine all the labels into a single label.
